eslint-warning-thresholds.json

```diff
@@ -27,79 +27,10 @@
     "@typescript-eslint/no-base-to-string": 1,
     "@typescript-eslint/no-shadow": 1
   },
```

Comment:
Eslint warning thresholds json file automatically updated to remove linting violations after running `yarn lint`